docusaurus/docs/dev-docs/migration/v4-to-v5/additional-resources/plugins-migration.md

```diff
@@ -1,35 +1,50 @@
 ---
-title: Plugins upgrade summary
+title: Plugins upgrade
```

Comment:
```suggestion
title: Plugins upgrade summary
```

I would recommend we keep the current title, to manage user's expectations. This is still very much a summary at this point.